triggers/tests/src/test/scala/com/digitalasset/daml/lf/engine/trigger/test/AbstractFuncTests.scala

```diff
@@ -319,7 +319,7 @@ abstract class AbstractFuncTests
         for {
           client <- ledgerClient(
             // Sufficiently low that the transaction is larger than the max inbound message size
-            maxInboundMessageSize = 300
+            maxInboundMessageSize = 100
```

Comment:
Why did this change?

Reply:
It was not failing for 300 for some Maybe the behavior with SoX produces lower chunks?

Reply:
hm don’t really understand why but no issue with changing it.